Makefile

```diff
@@ -488,3 +488,7 @@ dep-ui: test-tools-image
 
 dep-ui-local:
 	cd ui && yarn install
+
+.PHONY: cmd-docs
```

Comment:
I think we should add cmd-docs as part of `codegen-local`  https://github.com/argoproj/argo-cd/blob/master/Makefile#L159 . This way CI will ensure that docs are up to date